pkg/metadata/inventories/README.md

```diff
@@ -43,6 +43,14 @@ The payload is a JSON dict with the following fields
   - `version` - **string**: the version of the Agent.
   - `flavor` - **string**: the flavor of the Agent. The Agent can be build under different flavor such as standalone
     dogstatsd, iot, serverless ... (see `pkg/util/flavor` package).
+  - `config_apm_dd_url` - **string**: the configuration value `apm_config.dd_url`
```

Comment:
Could we add that password are scrubbed for those ?

Reply:
I did.  But, this doc is probably only of use to authors of the intake receiving this information.  Is there another spot we should write that down?

Reply:
I don't think so.